src/AWS.SessionProvider.Net35.csproj

```diff
@@ -24,18 +24,11 @@
   <PropertyGroup>
     <SignAssembly>true</SignAssembly>
   </PropertyGroup>
-  <Choose>
-    <When Condition=" '$(AWSKeyFile)' == '' ">
-      <PropertyGroup>
-        <AssemblyOriginatorKeyFile>awssdk.dll.snk</AssemblyOriginatorKeyFile>
-      </PropertyGroup>
-    </When>
-    <Otherwise>
-      <PropertyGroup>
-        <AssemblyOriginatorKeyFile>$(AWSKeyFile)</AssemblyOriginatorKeyFile>
-      </PropertyGroup>
-    </Otherwise>
-  </Choose>
+
+  <PropertyGroup>
+    <AssemblyOriginatorKeyFile>public.snk</AssemblyOriginatorKeyFile>
```

Comment:
why don't we have `<SignAssembly>true</SignAssembly>` here?

Reply:
It's there on line 25.